opentelemetry-auto-instrumentation/src/opentelemetry/auto_instrumentation/auto_instrumentation.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
```

Comment:
What is the advantage of using this script over setting `PYTHONPATH` manually? Consider also documenting that in the README.

Reply:
In fact, the advantage is not having to set up `PYTHONPATH` manually. Having to do so means bothering the end user to find where the directory of `sitecustomize.py` is and prepending its path to `PYTHONPATH` before running the command that could do so by itself.